### PR TITLE
Create primalityTest.go

### PR DESCRIPTION
**Pull request by vedantmamgain**

### CODE REVIEW

File: math/primalityTest.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//A primality test is an algorithm for determining whether an input number is prime.Among other fields of mathematics, it is used for cryptography.
+//Unlike integer factorization, primality tests do not generally give prime factors, only stating whether the input number is prime or not.
+//Source - Wikipedia https://en.wikipedia.org/wiki/Primality_test
+package main
```
Comment (nomisrevol) [inline math/primalityTest.go:4]: `primalityTest` is not a good name for this prime checking implementation. Please rename it to `prime_check.go` and implement a test file also.

File: math/primalityTest.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+func NaiveApproach(n int) bool {
+	if n==1{
```
Comment (nomisrevol) [inline math/primalityTest.go:8]: It is more strict with condition `n < 2`.

File: math/primalityTest.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+		if n%i == 0 {
+			return false
+		}
+	}
+	return true
+}
+
+func PairApproach(n int) bool {
+	if n==1{
```
Comment (nomisrevol) [inline math/primalityTest.go:21]: `n < 2` also.

File: math/primalityTest.go [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+		return false
+	}
+	for i:=2;i*i<=n;i++ {
```
Comment (nomisrevol) [inline math/primalityTest.go:24]: What will happen if `i * i` overflow?

**Review by nomisrevol — CHANGES_REQUESTED** [review]

**Comment by vedantmamgain** [comment]
Check it now. @nomisrevol

**Comment by nomisrevol** [comment]
@vedantmamgain How about test file?

**Comment by vedantmamgain** [comment]
@nomisrevol  Added test file for both the approaches.

**Comment by vedantmamgain** [comment]
@nomisrevol Is there anything that is remaining?

**Comment by nomisrevol** [comment]
@vedantmamgain It's ok for now. Sorry for the late reply. I will merge it, fyi.

**Pull request merged**